package.json

```diff
@@ -51,15 +51,15 @@
   "peerDependencies": {
     "react": "^16.6.1 || ^17.0.0 || ^18.0.0",
     "react-dom": "^16.6.1 || ^17.0.0 || ^18.0.0",
-    "styled-components": "^6.0.0"
+    "styled-components": "^5.1.0 || ^6.0.0",
+    "stylis": "^4.0.0"
```

Comment:
styled-components 6 lists stylis ^4.3.0 as a dependencies. Does this need to be a peer dependency? (especially since it isn't a dependency for styled-components 5)

Reply:
Good note, I'll remove stylis from the peer dependencies